.azure-pipelines/ultimate-pipeline.yml

```
@@ -193,44 +195,45 @@ stages:
       parameters:
         masterCommitId: $(masterCommitId)
     - template: steps/install-latest-dotnet-sdk.yml
+
     - script: tracer\build.cmd BuildTracerHome
       displayName: Build tracer home
 
     - script: tracer\build.cmd BuildNativeLoader
       displayName: Build monitoring home
 
-    - publish: $(tracerHome)
-      displayName: Upload Windows tracer home directory
-      artifact: windows-tracer-home
-
     - publish: $(System.DefaultWorkingDirectory)
       displayName: Upload working directory after the managed build
       artifact: build-windows-working-directory
 
-- stage: build_windows_profiler
-  dependsOn: [master_commit_id]
-  variables:
-    masterCommitId: $[ stageDependencies.master_commit_id.fetch.outputs['set_sha.master']]
-  jobs:
-  - template: steps/update-github-status-jobs.yml
-    parameters:
-      jobs: [build]
-
-  - job: build
-    pool:
-      vmImage: windows-2019
-    steps:
-    - template: steps/clone-repo.yml
-      parameters:
-        masterCommitId: $(masterCommitId)
-    - template: steps/install-latest-dotnet-sdk.yml
-
     - script: tracer\build.cmd BuildProfilerHome
       displayName: Build profiler home
 
-    - publish: $(System.DefaultWorkingDirectory)/profiler/_build/DDProf-Deploy
-      displayName: Upload Windows profiler home directory
-      artifact: windows-profiler-home
+    - script: tracer\build.cmd PackageTracerHome --ProfilerHome $(System.DefaultWorkingDirectory)/profiler/_build/DDProf-Deploy
+      displayName: Build MSI and Tracer home
+
+    - script: tracer\build.cmd PackageMonitoringHomeBeta
+      displayName: Build beta MSI
```

`PackageMonitoringHomeBeta` just calls `BuildMsi` so we may as well delete it now 🙂 
```suggestion
    - script: tracer\build.cmd PackageTracerHome --ProfilerHome $(System.DefaultWorkingDirectory)/profiler/_build/DDProf-Deploy
      displayName: Build MSI and Tracer home
```

We should also delete the `PackageMonitoringHomeBeta` target, and update the `PackageTracerHome` to

```csharp
Target PackageTracerHome => _ => _
    .Description("Builds NuGet packages, MSIs, and zip files, from already built source")
    .After(Clean, BuildTracerHome, BuildProfilerHome, BuildNativeLoader)
    .DependsOn(CreateRequiredDirectories)
    .DependsOn(ZipTracerHome)
    .DependsOn(BuildMsi)
    .DependsOn(PackNuGet);
```

---

Fixed in 5edb6ffac1b5d96e5201336a81a5e56193263dd1